extensions/amp-pan-zoom/0.1/amp-pan-zoom.js

```diff
@@ -324,6 +321,26 @@ export class AmpPanZoom extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Given a x offset relative to the viewport, return the x offset
+   * relative to the amp-pan-zoom component.
+   * @param {number} clientX
+   */
+  getOffsetX_(clientX) {
+    return (this.elementBox_.left - this.getViewport().getScrollLeft())
+      - clientX;
```

Comment:
mini-nit: inconsistent indentation with the function below

Reply:
The dangling `- variable` slows down the read through a little. Consider using an intermediate variable to store and name the `this.elementBox_.left - this.getViewport().getScrollLeft()` computation for clarity. A long line can be a symptom indicating that too much is happening in one line.